pkg/operator/controllers/machinehealthcheck/staticresources/mhcremediationalert.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
```

Comment:
Question about the PrometheusRule - [this section](https://docs.openshift.com/container-platform/4.10/monitoring/managing-alerts.html#creating-alerting-rules-for-user-defined-projects_managing-alerts) of openshift documentation says that we can't create new alerts in `openshift-*` namespaces. Is there something different about the strategy here that makes it work?

Reply:
I was able to create alerting rules with `oc create -f mhcremediationalert.yaml` just fine in the `openshift-machine-api` namespace and the alert fired without issue. I am open to changing the namespace to something more ARO specific to help indicate to the customer where the alert is from when it shows up in their management console.

Reply:
I was able to create alerting rules with `oc create -f mhcremediationalert.yaml` just fine in the `openshift-machine-api` namespace and the alert fired without issue. I am open to changing the namespace to something more ARO specific to help indicate to the customer where the alert is from when it shows up in their management console.